app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :likes
+  has_many :comments
```

Comment:
- [ ] As stated by the previous reviewer, you need to add foreign keys like in the ERD.  Look:

```ruby
has_many :posts, foreign_key: 'author_id'
has_many :comments, foreign_key: 'author_id'
has_many :likes, foreign_key: 'author_id'
```

Now that you know how to do it, go ahead and implement it! 💪